newrelic.js

```diff
@@ -0,0 +1,53 @@
+'use strict'
+/**
+ * New Relic agent configuration.
+ *
+ * See lib/config/default.js in the agent distribution for a more complete
+ * description of configuration variables and their potential values.
+ */
+exports.config = {
+  /**
+   * Array of application names.
+   */
+  app_name: ['proxy-eric'],
+  /**
+   * Your New Relic license key.
+   */
+  license_key: `${process.env.NEWRELIC_KEY}`,
```

Comment:
I had gitignore the file. But I like that you used an env variable here :) I may try to get mine to work with one as well.